plugins/janus_videoroom.c

```diff
@@ -87,7 +87,7 @@ room-<unique room ID>: {
 	notify_joining = true|false (optional, whether to notify all participants when a new
 				participant joins the room. The Videoroom plugin by design only notifies
 				new feeds (publishers), and enabling this may result extra notification
-				traffic. This flag is particularly useful when enabled with \c require_pvtid
+				traffic. This flag is particularly useful when enabled with require_pvtid
```

Comment:
Why did you remove the `\c` here?

Reply:
Because this text is verbatim already, and \c [prints like "\c require_pvtid"](https://janus.conf.meetecho.com/docs/videoroom.html)

Reply:
Oh right, good catch!